docs: Update documentation prior to version update (#122)
CCBR/SINCLAIR: docs/user-guide/getting-started.md

```
# Grab an interactive node
sinteractive --mem=64g --cpus-per-task=16 --time=8:00:00 --gres=lscratch:128
```

> ## 1.5 Installing and setting up SINCLAIR
> 
> The CCBRPipeliner module on Biowulf also loads module dependencies, and should be loaded prior to running SINCLAIR:
> 
> ```
> module load ccbrpipeliner
> ```
> 
> SINCLAIR can be downloaded directly from the CCBR Github page with the `git clone` command:
> 
> ```
> git clone https://github.com/CCBR/SINCLAIR
> ```
> 
> SINCLAIR can also be initialized on Biowulf using the ccbrpipeliner module (as of ccbrpipeliner version 8):
> 

Reviewer: Using the ccbrpipeliner module is the recommend way to run sinclair. We can have a separate guide for advanced users who may wish to use a dev version or install in other environments.

````suggestion
## 1.5 Installing and setting up SINCLAIR

SINCLAIR is installed on Biowulf and can be loaded using the ccbrpipeliner module (as of ccbrpipeliner release 8):

```
module load ccbrpipeliner/8
```

````

Author: I'm trying to get it up so that another CCBR user is able to use it this week. Is there a way we can table this change for the full ccbrpipeliner/8 upload?

Reviewer: > I'm trying to get it up so that another CCBR user is able to use it this week. Is there a way we can table this change for the full ccbrpipeliner/8 upload?

sure, in that case you should be sure to mention that instead of just `sinclair run` it will be `path/to/clone/bin/sinclair run`

```

> sinclair init

Reviewer: Let's recommend using the `--output` option
```suggestion
sinclair init --output /path/to/output/dir
```
